config/ec2/config.go

```diff
@@ -260,6 +260,11 @@ func Configure(p *config.Provider) {
 		}
 	})
 
+	p.AddResourceConfigurator("aws_vpc_peering_connection_accepter", func(r *config.Resource) {
```

Comment:
Could you add a working example?

Reply:
Updated the PR with a basic example, thanks @muvaf